tests/Makefile

```diff
@@ -53,47 +53,52 @@ BINS=\
 	getid \
 	setid
 
+FORCE:
+.PHONY: all clean run gen_expected verify FORCE
+
 all: $(BINS)
 
 clean:
-	rm -f $(BINS) *.out
+	@$(RM) -v $(BINS) *.out
+	@$(RM) -rv gen/
+run: $(BINS:%=%.run)
```

Comment:
Why, this would trigger a rebuild before compiling, ie. ifI aborted compilation and want to clean up, I first need to recompile in order to delete the just created files.

Reply:
Ah, then I think you need to either update `ci.sh` or update the root Makefile to run `$(MAKE) -C tests all run` for the `test` target right?

Reply:
Ah, I misread, I thought you meant `clean` should depend on `all` because that's how it looked when viewed in mail, my bad.  But still it shouldn't depend because I moved that logic to the individual `%.run` target, ie. `run` depends on `atof.run atoi.run ...` where each of those depends on the respective binary `atof`, `atoi`, ... -- which then are built if needed.  However make complains about no rule to build target `atof.run` which clearly exists, and I'd say all dependencies too. And I've cloned the whole thing cleanly here and ran ci.sh -- and it built.  Honestly no idea what went wrong in travis.

Reply:
Ah, the issue appears to be with older versions of `make`. I updated the `.travis.yml` hopefully this fixes travis :crossed_fingers: